ReactAndroid/src/main/java/com/facebook/react/views/art/ARTShapeShadowNode.java

```diff
@@ -164,6 +170,46 @@ protected boolean setupStrokePaint(Paint paint, float opacity) {
     return true;
   }
 
+  /*
+   * clculate stops count from array
+   */
+  private static int getStopsCount(float[] value) {
+    int values = (value.length - 5) / 4;
+    int count =  values - (values/5);
```

Comment:
And then you also divide by 5.  I don't understand why this works.  Maybe in the comment explain what the array represents and what you are returning.